Automattic/vip-go-mu-plugins
# Update EP fork

## Description

- Force fetching ep_wpcli_sync_interrupted transient from remote to allow for more reliable remote interruption https://github.com/Automattic/ElasticPress/pull/108
- Fix indexing blocking multisite check https://github.com/Automattic/ElasticPress/pull/102

## Changelog Description

### Plugin Updated: Enteprise Search

- Remove network-wide lock as it prevents indexing of separate subsites at the same time - https://github.com/Automattic/ElasticPress/pull/102
- Force fetching ep_wpcli_sync_interrupted transient from remote to allow for more reliable remote interruption https://github.com/Automattic/ElasticPress/pull/108

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Try starting an index on the subsite.
2. Try starting an index on another subsite.
3. Observe both are working. 
4. Try `wp vip-search stop-indexing --url=subsite1` and make sure the indexing for subsite1 stopped but the other one works
